OWNERS

```diff
@@ -1,16 +1,11 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- dims
-- derekwaynecarr
-- dchen1107
-- Huang-Wei
-- sig-node-api-approvers
-- api-approvers
+  - swatisehgal
+  - @ffromani
```

Comment:
can we please omit the leading `@` please? (here and everywhere else)

Reply:
Ah, right!

Reply:
Done, Thanks!